integration-testing/test/conftest.py

```diff
@@ -11,6 +11,12 @@ def pytest_addoption(parser):
     parser.addoption(
         "--peer-count", action="store", default="2", help="number of peers in the network (excluding bootstrap node)"
     )
+    parser.addoption(
+        "--start-timeout", action="store", default="0", help="timeout in seconds for starting a node. Defaults to 30 + peer_count * 10"
```

Comment:
Maybe make the default equal to 30 and remove the if statement below?

Reply:
if there is no value provided the formula `start-timeout = 30 + peer-count * 10` is used. If a value is provided then we use that exact value